### PR TITLE
add an association between tables, add list- model, routes, service, …

### PR DESCRIPTION
**Pull request by lisperata**
…controller

image
image
image
image
image
image

### CODE REVIEW

File: contact/contactController.ts [hunk 1]
```diff
@@ -12,9 +12,9 @@ export const postContact = async (
     await contact.addContactWithSequelize();
     //with query
     //await contact.addContact();
```
Comment (CatDidenko) [inline contact/contactController.ts]: Remove comments

File: list/listService.ts [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+  }
+
+  public async addNewList(): Promise<void> {
+    const name = this.name;
+    await listModel.create({ name });
+  }
+
+  public async addContactToList(): Promise<void> {
+    const uuidOfContact: string = this.uuid;
+    const nameOfList: string = this.name;
```
Comment (CatDidenko) [inline list/listService.ts]: How uuid of List class related to Contact uuid?
this.uuid should contain uuid of the List.

File: list/listService.ts [hunk 3]
```diff
@@ -1,12 +1,35 @@
...
+  static async getContactsOfListByListId(listId: string): Promise<string> {
+    return await Contact.getContactsOfList(listId);
+  }
+
+  public async addNewList(): Promise<void> {
+    const name = this.name;
+    await listModel.create({ name });
+  }
+
+  public async addContactToList(): Promise<void> {
```
Comment (CatDidenko) [inline list/listService.ts]: You should provide contact id in this function and add it to the list

**Review by CatDidenko — CHANGES_REQUESTED** [review]

File: list/listService.ts (hunk 3 above)
Comment (lisperata) [reply list/listService.ts]: The values were provided as parameters to the function

File: list/listService.ts (hunk 2 above)
Comment (lisperata) [reply list/listService.ts]: constructor was deleted, the values were provided as parameters to the function

File: contact/contactController.ts (hunk 1 above)
Comment (lisperata) [reply contact/contactController.ts]: Removed

File: contact/contactController.ts [hunk 4]
```diff
@@ -24,15 +20,10 @@ export const putContact = async (
...
 ): Promise<void> => {
-  const contact = new Contact(name, email, uuid);
   try {
-    //with model.update
-    await contact.changeContactByIdWithSequelize();
-    //with query
-    //await contact.changeContactById();
-
-    res.type("plain").send("Data has been successfully changed");
+    await Contact.changeContactByIdWithSequelize(name, email, uuid);
```
Comment (CatDidenko) [inline contact/contactController.ts]: Remove sequelize from name

**Review by CatDidenko — APPROVED** [review]

**Pull request merged**